app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -285,6 +286,23 @@ export default function ExploreDisplayTabs({
                 />
               </div>
             }
+            { enabledTabs.includes('correlatedScatter') &&
+              <div className={shownTab === 'correlatedScatter' ? '' : 'hidden'}>
+                <ScatterPlot
+                  studyAccession={studyAccession}
+                  {...exploreParams}
+                  isCorrelatedScatter={true}
+                  dimensions={getPlotDimensions({
+                    numColumns: 1,
+                    numRows: 1,
+                    hasTitle: true,
+                    showRelatedGenesIdeogram: false
+                  })}
+                  isCellSelecting={isCellSelecting}
+                  plotPointsSelected={plotPointsSelected}
+                />
```

Comment:
All [examples of gene correlation plots](https://www.google.com/search?q=gene+correlation+plot) I've seen -- except for a sketch from the original SCP feature request in https://github.com/broadinstitute/single_cell_portal/issues/38 -- have numerical tick marks / gridline labels for the X and Y axes, but your helpful screenshot lacks them.

I suspect we need tick marks for scientific quality here, but I defer to @jlchang on their priority -- e.g. blocker for MVP, ticket-worthy for next sprint, or a later opportunity for improvement.

Reply:
Yes we should have tick marks but I don't consider the lack to be blocking as long as we intend to add them in a timely manner.

Reply:
cool, I can't imagine adding tickmarks in plotly will be too hard

Reply:
added